File: Programação/C/cara coroa.cpp
#include <stdio.h>

int main() {
    int N, i, total, X, Y;
        scanf("%i", &N);
        i = 0;
        X = 0; 
        Y = 0;       
        while (i < N) {
            scanf("%i", &total);
            
            if (total == 0){ 
                X++;
            }else{
                Y++;
            }
            i++;
        }
        
        printf("Mary won %i times and John won %i times\n", X, Y);
}

File: Programação/C/cara coroa.py
n = int(input(''))
m = 0
j = 0
while n != 0:
    for i in range (n):
        moeda = int(input(''))
        if moeda == 1:
            m+= 1
        else:
            j += 1

print("Mary won {} times and John won {} times" .format(m, j))